src/views/Media/NewsletterSignup.vue

```diff
@@ -0,0 +1,232 @@
+<template>
+  <div id="cve-secondary-page-main-container">
+    <div class="columns is-centered">
+      <div class="column is-12-desktop is-12-tablet cve-main-column-content-width">
+        <main id="cve-main-page-content" role="main">
+          <div class="content">
+            <h1 class="title">Email Newsletter</h1>
+            <h2 class="title">Subscribe to the CVE<sup>®</sup> Program Newsletter</h2>
+            <p class="content">
+              <span class="has-text-weight-bold">CVE-Announce</span> provides general news about CVE such as new CNAs, new
+              website features, CVE in the news, etc. Messages are sent infrequently, once a
+              month or less.
+            </p>
+            <div v-if="!submitted" class="cve-white-bg-gray-border-container">
+              <form id="emailSignup" class="cve-top-margin" @submit="checkForm" novalidate="true">
+                <input type="hidden" name="X" value="9">
+                <input type="hidden" name="Y" value="4">
+               <div v-if="errors.length" class="notification is-danger">
+                 <p id="dangerIcon" class="is-hidden">danger</p>
+                   <font-awesome-icon style="flex: 0 0 40px;" size="1x"  icon="exclamation-circle" role="danger"
+                    aria-labelledby="dangerIcon" aria-hidden="false" />
+                 <b> Please correct the following error(s):</b>
```

Comment:
To be consistent, please use the `has-text-weight-bold` class. E.g.: `<p class="has-text-weight-bold">...`